src/gui/systray.cpp

```diff
@@ -360,15 +360,15 @@ QPoint Systray::computeWindowPosition(int width, int height) const
 
 QPoint Systray::calcTrayIconCenter() const
 {
-    // QSystemTrayIcon::geometry() is broken for ages on most Linux DEs (invalid geometry returned)
-    // thus we can use this only for Windows and macOS
-#if defined(Q_OS_WIN) || defined(Q_OS_MACOS)
-    auto trayIconCenter = geometry().center();
-    return trayIconCenter;
-#else
-    // On Linux, fall back to mouse position (assuming tray icon is activated by mouse click)
-    return QCursor::pos();
-#endif
+    QRect iconRect = geometry();
```

Comment:
bah stupid me, only realize it now... minor nitpick: `const auto iconRect`?

Reply:
Why stupid? I think it's quite natural in reviewing to find some issues later.